### PR TITLE
Feat(107): Participants section

### LINKED ISSUE #107: Participants section
## Overview

The participants block on the proposal view page needs to be completed to match the designs

### PR DESCRIPTION
**Pull request by RyRy79261**
## Overview

This PR implements a scroll area component from Shadcn and fixes some styling inconsistencies

## Review guide

It's tricky to test this out however in implementing I loaded a bunch of dummy elements into the participants section element

## Related PR 

- closes #107

### CODE REVIEW

File: src/components/create-proposal/create-proposal-wallet-step/create-proposal-wallet-step.component.tsx [hunk 1]
```diff
@@ -35,7 +35,11 @@ const CurrentFormStep = ({ formStep }: { formStep: WalletStepEnum }) => {
           </p>
           <p className="font-size-x4 line-height-x5 place-self-start">
             You can purchase MENTO{" "}
-            <a href={"https://app.mento.org"} target="_blank">
+            <a
```
Comment (denviljclarke) [inline src/components/create-proposal/create-proposal-wallet-step/create-proposal-wallet-step.component.tsx:38]: nit - Lets use next Link

File: src/components/_shared/progress-bar/progress-bar.component.tsx [hunk 2]
```diff
@@ -120,36 +119,78 @@ export const ProgressBar = ({
...
-              )}
-              style={{
-                width: `${progress}%`,
-                color,
-                zIndex: 10 - index,
-              }}
-            ></div>
-          );
-        })}
+      {/* <div className="flex h-4 w-full overflow-hidden rounded-3xl  border-[0.5px] border-solid border-black dark:border-gray"> */}
```
Comment (denviljclarke) [inline src/components/_shared/progress-bar/progress-bar.component.tsx:176]: Do we still need this?

File: src/components/proposal-current-votes/proposal-current-votes.component.tsx [hunk 3]
```diff
@@ -1,148 +1,153 @@
...
+                        {NumbersService.parseNumericValue(
+                          formatUnits(quorumNeeded || 0n, 18),
+                        )}
+                      </>
+                    )}
+                    {quorumNeeded <= BigInt(proposal.votes.for.total) && (
+                      <>Quorum reached.</>
+                    )}
+                  </>
+                )}
```
Comment (denviljclarke) [inline src/components/proposal-current-votes/proposal-current-votes.component.tsx:150]: nit - extra div in this file line 155

File: src/styles/brand.scss [hunk 4]
```diff
@@ -257,6 +257,8 @@ $themes: (
 }
 
 @mixin default-input-error-message {
+  // "p2-1 text-sm font-semibold text-error-dark"
```
Comment (denviljclarke) [inline src/styles/brand.scss:260]: what's left to do before we can kill these scss files?

File: src/components/proposal-current-votes/proposal-current-votes.component.tsx [hunk 5]
```diff
@@ -1,148 +1,153 @@
...
-              <div
-                className={cn(
-                  "flex h-x3 w-x3 items-center justify-center border border-black text-white",
-                  "bg-black",
-                  "rounded-[4px]",
+          <div>
+            <div className="flex items-center justify-start gap-x3">
+              {!!quorumNeeded &&
+                quorumNeeded > BigInt(proposal.votes.for.total) && (
+                  <CrossMarkIcon />
```
Comment (denviljclarke) [inline src/components/proposal-current-votes/proposal-current-votes.component.tsx:122]: invert colors in dark mode

File: src/components/proposal-current-votes/proposal-current-votes.component.tsx [hunk 6]
```diff
@@ -1,148 +1,153 @@
...
+                          formatUnits(proposal.votes.for.total, 18),
+                        )}{" "}
+                        of{" "}
+                        {NumbersService.parseNumericValue(
+                          formatUnits(quorumNeeded || 0n, 18),
+                        )}
+                      </>
+                    )}
+                    {quorumNeeded <= BigInt(proposal.votes.for.total) && (
+                      <>Quorum reached.</>
```
Comment (denviljclarke) [inline src/components/proposal-current-votes/proposal-current-votes.component.tsx:147]: nit - remove period after reached

**Review by denviljclarke — CHANGES_REQUESTED** [review]
Good work! Everything looks good, I left a few small style related comments

File: src/components/proposal-current-votes/proposal-current-votes.component.tsx (hunk 6 above)
Comment (RyRy79261) [reply src/components/proposal-current-votes/proposal-current-votes.component.tsx:147]: @denviljclarke Where is the the confirmed behaviour?

File: src/components/proposal-current-votes/proposal-current-votes.component.tsx (hunk 5 above)
Comment (RyRy79261) [reply src/components/proposal-current-votes/proposal-current-votes.component.tsx:122]: Is this confirmed? It looked like it worked in dark mode

**Review by nvtaveras — APPROVED** [review]
LGTM functionality wise, not a frontend expert so won't comment on the implementation details, although I noticed that there were some changes unrelated to the ticket which would be good to keep separate in in the future when possible.

Also curious if you tested it with a proposal with lots of votes in the different sections?

**Comment by RyRy79261** [comment]
@nvtaveras So I tested with a 50/50 `for` & `against` and the abstain is just the blank area presently

**Review by denviljclarke — APPROVED** [review]
LGTM

**Pull request merged**